sass/components/ClusterOverview/consumers.scss

```diff
@@ -1,10 +1,51 @@
-.kubevirt-consumers__bar .progress {
+.kubevirt-consumers__bar {
+  height: 20px;
```

Comment:
the em units are preferable in other places as well. We should probably convert the whole dashboard to be consistent with this.

Reply:
fixed